pkg/loadbalancer/tasks.go

```diff
@@ -138,6 +140,58 @@ func (u *UpdateHosts) Execute(runtime connector.Runtime) error {
 	return nil
 }
 
+type GetInterfaceName struct {
+	common.KubeAction
+}
+
+func (g *GetInterfaceName) Execute(runtime connector.Runtime) error {
+	host := runtime.RemoteHost()
+	cmd := fmt.Sprintf("ip addr "+
+		"| grep -B 2 %s "+
+		"| head -n 1 "+
+		"| awk -F: '{ print $2 }' "+
+		"| tr -d [:blank:]", host.GetAddress())
+	mes, err := runtime.GetRunner().SudoCmd(cmd, false)
+	interfaceName := strings.Split(mes, ":")[1]
```

Comment:
What happened if the command result `mes` is not your expect case? 
For example:
<img width="844" alt="image" src="https://user-images.githubusercontent.com/43993589/181876333-e4fa470e-9899-41a6-a407-e2981810cf99.png">
The use the `strings.Split("eth0", ":")[1]`, the program will be panic.